src/lib/specialistBios/data.json

```diff
@@ -0,0 +1,137 @@
+[
+  {
+    "firstName": "Jessica",
+    "name": "Jessica Backus",
+    "email": "jessica@artsymail.com",
+    "jobTitle": "VP, Collector Sales",
+    "specialty": "collectorServices",
+    "bio": "An early Artsy employee, Jessica has nearly 2 decades of experience in art and tech. Driven by a passion to help people connect with art they love, Jessica has launched multiple strategic initiatives to evolve and grow the art market. She began her career working with clients in Berlin at the gallery Peres Projects and worked at Phillips before joining Artsy. She holds a master’s degree in art history from Hunter and a bachelor’s degree from Columbia University.",
+    "imageUrl": "https://files.artsy.net/images/Jessica-Mobile.webp"
+  },
+  {
+    "firstName": "Christine",
```

Comment:
**question**: Could `firstName` be computed reliably from `name`? We have some similar logic for [splitting names on the backend](https://github.com/artsy/gravity/blob/a319430dadefbcb7bd850a748d19d6a96a0e092f/app/models/domain/artist.rb#L277-L286) we could borrow from.

Reply:
thanks, done!